lib/slug/slug.rb

```diff
@@ -36,63 +36,66 @@ def slug source, opts={}
         with: /\A[a-z0-9-]+\z/,
```

Comment:
The diff on this file is much clearer when viewed without the white space changes. We indented several methods within a module.